test/extensions/stats_sinks/open_telemetry/open_telemetry_integration_test.cc

```diff
@@ -181,6 +193,31 @@ TEST_P(OpenTelemetryGrpcIntegrationTest, BasicFlow) {
   sendRequestAndWaitForResponse(request_headers, 0, default_response_headers_, 0);
   ASSERT_TRUE(waitForMetricsRequest());
 
+  switch (clientType()) {
+  case Grpc::ClientType::EnvoyGrpc:
+    test_server_->waitForGaugeEq(getFullStatName("cluster.otlp_collector.upstream_rq_active"), 0);
```

Comment:
This should be refactored as a utility method rather than copy+paste.

Reply:
Done, thanks for reviewing